### DISCUSSION TITLE
"setKerning()" method?

### DISCUSSION THREAD

**Discussion by oldmanwerther** [appcelerator/titanium_mobile#13068] [Q&A]
I'm making updates to an app (that I didn't build -- originally used Titanium SDK 8.3.1 -- but now updating to 10.0.2) and setKerning() method is used all over the .JS...  I'm not seeing any documentation if this was a Titanium specific method? or perhaps a custom method.

**dateLabel.setKerning is not a function.**

**Comment by jquick-axway** [comment]
Getter/setter property methods have been removed as of Titanium 10.0.0 in favor of using the property itself. You should have been getting deprecation warning in the log about this in older Titanium versions.

The only kerning feature that the Titanium SDK offers that I'm aware of is via attributes strings using the `Ti.UI.ATTRIBUTE_KERN` type.
https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_How-tos/User_Interface_Deep_Dives/Attributed_Strings.html#kerning

Perhaps the `setKerning()` method you were using came from a 3rd party module? If so, trying doing a `.kering = ` property assignment instead.

**Comment by m1ga** [comment]
You might have used this custom code: https://gist.github.com/rossmd/5904932 
But since that creates an attributed string you could switch to that or add that function back to your SDK (be careful with updates then)
